migration/db-id/migration.sh

```diff
@@ -0,0 +1,44 @@
+#! /bin/bash
+
+set -e
+
+if [ -z "$DB_ID_MIGRATION_REPOSITORY" ] ; then
+  export DB_ID_MIGRATION_REPOSITORY=https://raw.githubusercontent.com/eventuate-foundation/eventuate-common
+fi
+
+get_db_id_migration_path () {
+  search="eventuateCommonVersion="
+  version_line="$(grep $search ./gradle.properties)"
+  version=${version_line#$search}
+
+  if [[ $version =~ "BUILD-SNAPSHOT" ]]; then
+     version=master
+  fi
+
+  echo "${DB_ID_MIGRATION_REPOSITORY}/${version}"
+}
+
+if [ "${DATABASE}" == "mysql" ]; then
+  curl -s $(get_db_id_migration_path)/mysql/4.initialize-database-db-id.sql &> /dev/stdout | ./mysql-cli.sh -i
+elif [ "${DATABASE}" == "postgres" ]; then
+  curl -s $(get_db_id_migration_path)/postgres/5.initialize-database-db-id.sql &> /dev/stdout | ./postgres-cli.sh -i
+elif [ "${DATABASE}" == "mssql" ]; then
+  rm -rf migration
```

Comment:
This script cannot be run on this project because it will delete the `migration` dir.
Change the name of the temporary directory, e.g `tmp-migration`